languages/ocaml.scm

```diff
@@ -1081,7 +1081,7 @@
 (parenthesized_expression
   .
   "(" @append_empty_softline
-  (product_expression) @prepend_indent_start @append_indent_end
+  _ @prepend_indent_start @append_indent_end
   ")" @prepend_empty_softline
   .
 )
```

Comment:
The way the indentation is done (i.e., set and unset on each node between the parentheses) is a little unusual. It may be simpler to understand as:
```scm
(parenthesized_expression
  .
  "(" @append_empty_softline @append_indent_start
  ")" @prepend_empty_softline @prepend_indent_end
  .
)
```

Although, admittedly, this doesn't always do what you'd expect...so maybe you tried this already.

Reply:
I don't remember why I had it that way, but your solution works, and is more elegant. I've amended the PR